build/compose.yml

```diff
@@ -30,9 +30,8 @@ services:
     build: 
       context: ..
       dockerfile: build/node.dev.Dockerfile
-    env_file:
-      - path: ../dev.env
-        required: false
+    env_file: ../dev.env
```

Comment:
maybe we should rename this to not `dev` if it's in the actual release

Reply:
I think this is fine for now, this compose file is only really used for development currently. Definitely worth a look later how we get critical configuration into the node when things are more mature.